docs/USER_GUIDE.md

```diff
@@ -305,8 +316,13 @@ To add more users to the application:
 ### Making authenticated API requests
 
 To use the API directly, you will need to authenticate requests using the User
```

Comment:
I don't think the bit about needing to use the user pool is correct given IAM is an option

Reply:
Agreed